src/main/resources/logback-spring.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<configuration>
+    <appender name="Console"
+              class="ch.qos.logback.core.ConsoleAppender">
+        <layout class="ch.qos.logback.classic.PatternLayout">
+            <Pattern>
+                %X{startTime} %X{testName}: %highlight(%X{testStatus}) %X{exceptionMessage} %X{throwable}%ex%n
```

Comment:
✏️ I think adding the class name would really improve the quality of life.
I think it's something like
```suggestion
                %X{startTime} %class{0}#%X{testName}: %highlight(%X{testStatus}) %X{exceptionMessage} %X{throwable}%ex%n
```

Reply:
Word of warning from the logback docs: 

> Generating the caller class information is not particularly fast. Thus, its use should be avoided unless execution speed is not an issue.